collector/pg_database.go

```diff
@@ -53,12 +53,22 @@ var (
 		"Disk space used by the database",
 		[]string{"datname"}, nil,
 	)
+	pgDatabaseConnectionLimitsDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(
+			namespace,
+			databaseSubsystem,
+			"connection_limit",
+		),
+		"Connection limit set for the database",
+		[]string{"datname"}, nil,
+	)
 
-	pgDatabaseQuery     = "SELECT pg_database.datname FROM pg_database;"
-	pgDatabaseSizeQuery = "SELECT pg_database_size($1)"
+	pgDatabaseQuery                 = "SELECT pg_database.datname FROM pg_database;"
```

Comment:
Why not simplify this and get it all in one query?

```suggestion
	pgDatabaseQuery                 = "SELECT pg_database.datname, pg_database.datconnlimit FROM pg_database;"
```

Reply:
@SuperQ I kept the split to respect the way it was done and to be able to only retrieve connection limits for databases that were not excluded as it's currently done for the Database size. Should I still include the excluded databases then?

Reply:
I think the database size thing is done that way because it can't be gathered directly from the `pg_database` table.

Reply:
Yes, you're right. I've updated my pr with a fix for that.